README.md

```diff
@@ -1,34 +1,72 @@
-# ubuntu-image: build Ubuntu images
+# ubuntu-image
 
 [![ubuntu-image](https://snapcraft.io/ubuntu-image/badge.svg)](https://snapcraft.io/ubuntu-image)
 ![Build](https://github.com/canonical/ubuntu-image/actions/workflows/build-and-test.yml/badge.svg)
 [![codecov](https://codecov.io/gh/canonical/ubuntu-image/branch/main/graph/badge.svg?token=F9jE9HKo1a)](https://codecov.io/gh/canonical/ubuntu-image)
 [![Go Report Card](https://goreportcard.com/badge/github.com/canonical/ubuntu-image)](https://goreportcard.com/report/github.com/canonical/ubuntu-image)
 
+ubuntu-image is a tool used for generating bootable images. You can use it to build Ubuntu images such as:
 
-ubuntu-image is used to build the following Ubuntu images:
+- Snap-based Ubuntu Core images from model assertions
 
-* Snap-based Ubuntu Core images from model assertions
-* Classical preinstalled Ubuntu images using image definition
+- Classical preinstalled Ubuntu images using image definitions
 
-Future versions should be generalized to build more (eventually all) Ubuntu images.
+The future versions of this tool will be more generalized, allowing users to build a wider range of Ubuntu images, including all future releases.
 
+## Getting started
 
-## Requirements
+### Requirements
 
-Ubuntu 18.04 (Bionic Beaver) is the minimum platform requirement, but Ubuntu 20.04 (Focal Fossa) or newer is recommended. All required third party packages are available in the Ubuntu archive.
+* Ubuntu 18.04 (Bionic Beaver) or newer (recommended: Ubuntu 20.04 (Focal Fossa) or newer)
 
+* Ability to install snaps ([SnapStore: ubuntu-image](https://snapcraft.io/ubuntu-image))
+
+### Quickstart
+
+See [Build your first Ubuntu Core image](https://ubuntu.com/core/docs/build-an-image) for instructions on how to build an Ubuntu core image on a **Raspberry Pi**.
+
+### Building images
```

Comment:
Maybe use two sub-sections for the two modes of operation? Because in the final markdown render, it looks a bit hard to distinguish the two blocks of instructions.